packages/paste-website/src/pages/components/time-picker/index.mdx

```diff
@@ -0,0 +1,608 @@
+
+<LivePreview scope={{TimePicker, Label, HelpText, useUID, Anchor, Box, Combobox, formatReturnTime}} noInline>
+  {OnChangePicker}
+</LivePreview>
+
+## States
+
+### TimePicker with inline error
+
+Change a Time Picker to its error state and add an inline error if the value entered doesn't pass validation requirements.
+
+An inline error is placed below the field to inform a user of any errors in their value. If Help Text exists, the error text should replace and repeat the Help Text.
+
+<LivePreview scope={{TimePicker, Label, HelpText, useUID, Anchor}} noInline>
+  {HasErrorPicker}
+</LivePreview>
+
+### Disabled input
+
+Use a disabled Time Picker to show users that they can't interact with it.
+
+If you want to show information that can't be edited, use a [read-only](/components/input#read-only-input) field or consider another way of showing static information.
+
+<LivePreview scope={{TimePicker, Label, HelpText, useUID, Anchor}} noInline>
+  {DisabledPicker}
+</LivePreview>
+
+### Read-only input
+
+Use a read-only Time Picker when a its value can't be changed, but users should be able to read or apply focus on it.
+
+<LivePreview scope={{TimePicker, Label, HelpText, useUID, Anchor}} noInline>
+  {ReadOnlyPicker}
+</LivePreview>
+
+## Composition notes
+
+A Time Picker must have at least a label and an input.
+
+<LivePreview scope={{TimePicker, Label, useUID}} noInline>
+  {LabelOnlyPicker}
+</LivePreview>
+
+<Callout>
+  <CalloutTitle as="h4">Quick note about Time Picker DOM methods</CalloutTitle>
+  <CalloutText>
+    There are three methods that come with the HTML picker: <inlineCode>stepUp()</inlineCode>,{' '}
+    <inlineCode>stepDown()</inlineCode> and <inlineCode>select()</inlineCode>. Usage of these methods is possible, but
+    we <strong>do not</strong> encourage it, as they may not be supported in future versions of the Time Picker. Use at
+    your own risk!
+  </CalloutText>
+</Callout>
+
+### Positioning Time Pickers
+
+When using multiple Time Pickers, stack them vertically with `$space-80` spacing between each field. To stack them, you can use either a <Anchor href="/primitives/box">Box</Anchor> or a <Anchor href="/layout/stack">Stack</Anchor> (as seen in the example below).
+
+<LivePreview scope={{TimePicker, Label, HelpText, useUID, Anchor, Box, Stack}} noInline>
+  {StackOfPickers}
+</LivePreview>
+
+Avoid placing multiple pickers on the same horizontal row to help make it easier to scan a page vertically. Use a <Anchor href="/layout/grid">Grid</Anchor> if you need to place them horizontally.
+
+### Labels and Help Text
+
+Labels should clearly describe the time value being requested. They should be concise and descriptive, not instructional. To do this:
+
+- Use Help Text to provide instruction if needed. For example, don't use "Enter the date you wish to receive your bill below" as label text. Instead, use "Billing date" as a label and "Your account will be automatically billed on the above date." as Help Text.
```

Comment:
More of a general question but: how useful is this section now that we'll have docs for Help Text and Label? Is it redundant in a reinforcement way or a "this is too much" way?

Reply:
Yeah now that we have those pages it's definitely a little redundant, but my vote would be to keep it in both places because I imagine most people using time picker will read only the time picker docs (if that), not also label/help text